bpf/headers/bpf_helpers.h

```diff
@@ -137,6 +137,9 @@ enum libbpf_pin_type {
     LIBBPF_PIN_NONE,
     /* PIN_BY_NAME: pin maps by name (in /sys/fs/bpf by default) */
     LIBBPF_PIN_BY_NAME,
+
+    // XXX custom
+    LIBBPF_PIN_INTERNAL = 100
```

Comment:
I have decided to do this mainly so that we don't introduce a header that will need to be included everywhere, containing a single constant define.

I reckon this should be refactored if or when we decide to resplit the tracers.

Reply:
I'm not sure if i understand what are you saying here. What do you mean resplit the tracers?

Reply:
I'd add this explanation to a comment on LIBBPF_PIN_INTERNAL so we don't forget the context it in the future

Reply:
I think we need to move this into a header of our own, since we might copy - paste a new version of libbpf headers here and cause an compile error. To the person that does this it may not be obvious what caused the build break.

Reply:
@marctc I was under the impression everything was grouped together so that we could share the maps, but perhaps I am missing a part of the history. I intend to split the `tc` programs off `generic_tracer` to allow them to be loaded separately - it will be possible to do the same now for other programs if we conclude it's desirable.

@grcevski @mariomac I will move this constant elsewhere to avoid confusion.